app/Services/Model/AttendeeService.php

```diff
@@ -403,66 +405,31 @@ public function send(int $summit_id, array $payload, Filter $filter = null): voi
             return $this->attendee_repository->getAllIdsByPage(new PagingInfo(1, PHP_INT_MAX), $filter);
         });
 
-        foreach ($ids as $attendee_id)
+        if(!count($ids)){
```

Comment:
please review the strategy applied here 
https://github.com/OpenStackweb/summit-api/blob/8f248b27a8201e60adc0e2b2ca84b6ab93d25ea2/app/Services/Model/Imp/SummitRegistrationInvitationService.php#L408
an apply the same pagination methodology